cmd/lotus-miner/sealing.go

```diff
@@ -365,6 +365,12 @@ var sealingAbortCmd = &cli.Command{
 	Name:      "abort",
 	Usage:     "Abort a running job",
 	ArgsUsage: "[callid]",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "requestId",
```

Comment:
nit: cli flags usually don't use camelCase.

I'd call this flag `sched`, indicating that we want to cancel a scheduler request. `request-id` doesn't really sound like something related to the scheduler more to a request on a worker.